Over21/ViewController.swift

```diff
@@ -67,42 +73,57 @@ class ViewController: UIViewController {
 }
 
 extension ViewController: CaptureHelperDelegate {
-
+    
 }
 
 extension ViewController: CaptureHelperDeviceManagerPresenceDelegate {
```

Comment:
If we don't plan to use the D600 in this application, we can remove this delegate.

Reply:
Removed in latest commit.

Reply:
Did you push it because I don't see it?

Reply:
Yep, pushed 15 minutes ago.

Reply:
I think it's still there (the device manager notifications)...